sirepo/template/opal.py

```diff
@@ -222,6 +222,60 @@ def __init__(self):
         super().__init__(SIM_TYPE, self._FIELD_MAP)
 
     def to_madx(self, data):
+
+        def _get_len_by_id(data, id):
+            for e in data.models.elements:
+                if e._id == id:
+                    return e.l
+
+        def _get_element_type(data, id):
+            for e in data.models.elements:
+                if e._id == id:
+                    return e.type
+
+        def _get_drift(distance):
+            for e in data.models.elements:
+                if e.l == distance and e.type == 'DRIFT':
+                    return e
+            return False
+
+        def _insert_drift(distance, beam_idx, items_idx, pos, length):
+            d = _get_drift(distance)
+            n = LatticeUtil.max_id(data) + 1
+            m = 'D'+str(n)
```

Comment:
spacing

Reply:
``` m = 'D'+str(n)``` should be ``` m = 'D' + str(n)``` ?

Reply:
yes

Reply:
You can also do `m = f'D{n}' and python will convert to a string.